app/views/layouts/application.html.erb

```diff
@@ -11,10 +11,16 @@
   <div class="topnav">
     <%=link_to "All Pets", "/pets"%>
     <%=link_to "All Shelters", "/shelters"%>
+    <%=link_to "Favorites: #{pluralize(favorites.total_count, "pet")}", '/favorites' %>
```

Comment:
keeps count of favorited pets, but ran into weird bug where it did not or could not reset even after reset